code/modules/power/singularity/emitter.dm

```diff
@@ -1,6 +1,6 @@
 /obj/machinery/power/emitter
-	name = "Emitter"
-	desc = "A heavy duty industrial laser.\n<span class='notice'>Alt-click to rotate it clockwise.</span>"
+	name = "emitter"
+	desc = "A heavy-duty industrial laser used in containment fields and power generation.\n<span class='notice'>Alt-click to rotate it clockwise.</span>"

```

Comment:
Eeeeh, that sounds a little odd to me. I think something like "generally" or "often" used would sound better.